### PR TITLE
Remove phantomJS

### PR DESCRIPTION
**Pull request by gielow**

### CODE REVIEW

**Review by stoarca — CHANGES_REQUESTED** [review]

File: tests/spec/phantesta-spec.js [hunk 1]
```diff
@@ -211,7 +47,7 @@ describe('phantesta', function() {
       await page.quit();
       phantesta.destructiveClearAllSnapshots();
     }));
-    it('should test basic functionality', syncify(async function() {
+    fit('should test basic functionality', syncify(async function() {
```
[the remark this answers is not in the document]
Comment (gielow) [reply tests/spec/phantesta-spec.js]: Removed.

File: README.md [hunk 2]
```diff
@@ -105,10 +105,10 @@ a Remote Server along with the ability to view and review them. To deploy this r
 The remote server `optionally` takes in the following arguments:
    - `port` defaults to 3000
    - `savePath` is the path where screenshots uploaded will be saved in the remote host. Defaults to ~/phantesta
-   - `maxPhantestaServers` is the maximum number of phantesta server that will be run at any 
+   - `maxPhantestaServers` is the maximum number of phantesta server that will be run at any
```
[the remark this answers is not in the document]
Comment (gielow) [reply README.md]: Forgot to rebase on the latest.

File: README.md [hunk 3]
```diff
@@ -22,41 +22,41 @@ npm install --save-dev phantesta
...
-  afterEach(function() {
+    page = await createDriver();
+  }));
+
+  afterEach(syncify(async function() {
     phantesta.ungroup();
-  });
+
+    await page.quit();
+    phantesta.destructiveClearAllSnapshots();
```
[the remark this answers is not in the document]
Comment (gielow) [reply README.md:58]: Fixed.
Ps.: It was wrong on master.

[the remark this answers is not in the document]
Comment (stoarca) [reply README.md:58]: Doesn't look like this was fixed.

**Review by stoarca — APPROVED** [review]

**Pull request merged**